mappings/net/minecraft/client/render/chunk/ChunkBuilder.mapping

```diff
@@ -29,6 +29,9 @@ CLASS net/minecraft/class_846 net/minecraft/client/render/chunk/ChunkBuilder
 		ARG 2 glBuffer
 	METHOD method_22761 upload ()Z
 	METHOD method_22763 scheduleRunTasks ()V
+	METHOD method_34845 getToBatchCount ()I
+	METHOD method_34846 getChunksToUpload ()I
+	METHOD method_34847 getFreeBufferCount ()I
```

Comment:
Just curious, why not match the fields?

Reply:
These are named to match the strings mojang is using to describe these in the debug profiler.